Manifests/ManagedPreferencesApplications/nl.root3.support.plist

```diff
@@ -11,9 +11,9 @@
 	<key>pfm_domain</key>
 	<string>nl.root3.support</string>
 	<key>pfm_format_version</key>
-	<integer>1</integer>
+	<integer>1.1</integer>
```

Comment:
No, this is the actual manifest format version, which ftm should be `1`.  This is not what whould be updated, it's `pfm_version`.  Please review https://github.com/ProfileCreator/ProfileManifests/wiki/Manifest-Format if you have questions about what each of these do.